en/user_guide/media.rst

```diff
@@ -0,0 +1,60 @@
+Media Management
+================
+
+Open Orchestra allows to manage media in the media library panel
+
+Folder
+------
+
+All files are organised inside folders. It's possible to browse the folder tree to see files by clicking on the left panel.
+
+Create
+~~~~~~
+
+Creating a folder is done by clicking on "new folder" button in the left panel.
+Creation form need:
+* name: name of the folder
+* list of websites: list of websites using the folder content (multiple select with CTRL + Click)
+
+Remove
+~~~~~~
+
+To remove a folder, go in a folder and click on the trash button on the right.
+
+Files
+-----
+
+Manage files
+~~~~~~~~~~~~
+
+The add button in the folder's content allows to upload files.
+
+A trash icon appears on media preview hovering to deleting media.
+
+Click on media preview open a modal to crop files or add meta information.
+
+Crop
+~~~~
+
+Image crop offer some options :
+
+* original: keep original format
+* fixed height: adapt height and keep proportion
+* fixed width: adapt width and keep proportion
+* rectangle: reduce to fit in the rectangle format
+
+Once crop is done the preview is updated and some choices are added:
+
+* manual crop: to adjust your selection manually
+* upload an alternative image: to change image
```

Comment:
article missing
which image is changed if i upload an alternative image ? Is it the original or a thumbnail ? What happens to my cropping ?